build/helper/documentation_snippets.py

```diff
@@ -8,10 +8,10 @@
 '''
 
 rep_cap_attr_desc = '''
-This property can use repeated capabilities for {0}. If set or get directly on the
-{1}.Session object, then the set/get will use all repeated capabilities in the session.
-You can specify a subset of repeated capabilities using the Python index notation on an
-{1}.Session repeated capabilities container, and calling set/get value on the result.
+This property can be set/get to specific {1} within your {0}.Session object. Use Python index notation on the repeated capabilities container `{2}` to specify a subset.
+Example: `my_session.{2}[ ... ].{3}`,
```

Comment:
Do you know of a way to preview the class.rst locally as html, before I commit my changes?
I can open it as html once it's in github, even in the PR, but I'm not sure how to do it with just the locally generated file

Reply:
I found some tips online, but none of the extensions that I tried that would supposedly allow me to preview the file worked  properly (reStructuredText/sphinx, restview). I'm just going to push what I have and then use github to tell me if it looks good or not. It might require a few pushes to get it right. I apologize in advance for the noise.

Reply:
The html is generated when you build. You can find it in `generated\docs\html\nidcpower`.

Reply:
Ah, thanks for the tip. That helped a lot.

Reply:
`code-block` wasn't needed?

Reply:
I couldn't get it to format properly using the `code-block` tag. Maybe it was because it was nested under a <tip> block. I'm not sure. In any case, I found the :py:attr: and :py:meth: tags, and those seemed to have done the trick:

![image](https://user-images.githubusercontent.com/80292064/119555164-2976de00-bd63-11eb-8673-0ab16c2fc652.png)